opentelemetry-api/tests/context/test_contextvars_context.py

```diff
@@ -27,18 +28,19 @@
 
 
 def do_work() -> None:
-    context.set_current(context.set_value("say", "bar"))
+    context.attach(context.set_value("say", "bar"))
 
 
 class TestContextVarsContext(unittest.TestCase):
```

Comment:
could we create a base class of the context tests, which we can extend and set the context constructor? seems like a great way to ensure standard behavior across context implementations.

Reply:
done